<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QSpinBox::down-button{
    width: 50
}
QSpinBox::up-button{
    width: 50
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="TopButtonMenu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>991</width>
      <height>61</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="ConfigurationMenuPage">
     <widget class="QPushButton" name="MenuRestartConfigurationButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Restart Configuration</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="StandardMenu">
     <widget class="QPushButton" name="MenuHomeButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MenuConfigurationButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Configuration</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MenuRecepiesButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Recepies</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MenuStatisticsButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>0</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistics</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MenuSettingsButton">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="DMMPages">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>1001</width>
      <height>401</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="DrinkConfigPage">
     <widget class="QDoubleSpinBox" name="PumpAmount1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox
{
	font: bold 

}</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="prefix">
       <string>Amount Pump 1: </string>
      </property>
      <property name="suffix">
       <string> cl</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="PumpAmount2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox
{
	font: bold 

}</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="prefix">
       <string>Amount Pump 2: </string>
      </property>
      <property name="suffix">
       <string> cl</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="PumpAmount3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox
{
	font: bold 

}</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="prefix">
       <string>Amount Pump 3: </string>
      </property>
      <property name="suffix">
       <string> cl</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>300</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #rgb(68, 255, 16);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(2, 79, 0, 255), stop:0.19 rgba(50, 147, 22, 255), stop:0.275 rgba(236, 191, 49, 255), stop:0.39 rgba(243, 61, 34, 255), stop:0.555 rgba(135, 81, 60, 255), stop:0.667 rgba(121, 75, 255, 255), stop:0.825 rgba(164, 255, 244, 255), stop:0.885 rgba(104, 222, 71, 255), stop:1 rgba(93, 128, 0, 255));</string>
      </property>
      <property name="text">
       <string>Pour Drink</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>190</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Randomize drink</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Test1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test2</string>
       </property>
      </item>
     </widget>
     <widget class="QDoubleSpinBox" name="PumpAmount4">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>60</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox
{
	font: bold 

}</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="prefix">
       <string>Amount Pump 4: </string>
      </property>
      <property name="suffix">
       <string> cl</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="PumpAmount5">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>200</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox
{
	font: bold 

}</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="prefix">
       <string>Amount Pump 5: </string>
      </property>
      <property name="suffix">
       <string> cl</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="PumpAmount6">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>320</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox
{
	font: bold 

}</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="prefix">
       <string>Amount Pump 6: </string>
      </property>
      <property name="suffix">
       <string> cl</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PouringPage">
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="Pump1ProgressBar">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>50</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>251</width>
        <height>291</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::BottomToTop</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="Pump1ProgressBar_2">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>90</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="Pump1ProgressBar_3">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>130</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="Pump1ProgressBar_4">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>170</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="Pump1ProgressBar_5">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>210</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="Pump1ProgressBar_6">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>250</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::BottomToTop</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>300</y>
        <width>191</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>Abprt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>300</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <property name="text">
       <string>Temp text p1</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <property name="text">
       <string>Temp text p1</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>130</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <property name="text">
       <string>Temp text p1</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <property name="text">
       <string>Temp text p1</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>210</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <property name="text">
       <string>Temp text p1</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>240</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <property name="text">
       <string>Temp text p1</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ConfigGlassPage">
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>150</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm glass size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
